ci/scripts/github/common.sh

```diff
@@ -75,31 +71,15 @@ function update_conda_env() {
     # Deactivate the environment first before updating
     conda deactivate
 
-    ENV_YAML=${CONDA_ENV_YML}
-    if [[ "${MERGE_EXAMPLES_YAML}" == "1" || "${MERGE_DOCS_YAML}" == "1" ]]; then
-        # Merge the dev, docs and examples envs, otherwise --prune will remove the examples packages
-        ENV_YAML=${WORKSPACE_TMP}/merged_env.yml
-        YAMLS="${CONDA_ENV_YML}"
-        if [[ "${MERGE_EXAMPLES_YAML}" == "1" ]]; then
-            YAMLS="${YAMLS} ${CONDA_EXAMPLES_YML}"
-        fi
-        if [[ "${MERGE_DOCS_YAML}" == "1" ]]; then
-            YAMLS="${YAMLS} ${CONDA_DOCS_YML}"
-        fi
-
-        rapids-logger "Merging conda envs: ${YAMLS}"
-        conda run -n morpheus --live-stream conda-merge ${YAMLS} > ${ENV_YAML}
-    fi
-
-    if [[ "${SKIP_CONDA_ENV_UPDATE}" == "" ]]; then
-        rapids-logger "Checking for updates to conda env"
+    rapids-logger "Checking for updates to conda env"
 
-        # Remove default/conflicting channels from base image
+    # Remove default/conflicting channels from base image
+    if [[ -d /opt/conda/.condarc ]]; then
         rm /opt/conda/.condarc
+    fi;
```

Comment:
I dont think that this is necessary anymore and should probably be removed.

Reply:
This is necessary if you're running the CI container locally. It's why I added it.

Reply:
Just remove the whole if block and `rm` command